app/src/main/java/stage/hoogtebepaling5/Location/LocationClass.java

```diff
@@ -0,0 +1,199 @@
+package stage.hoogtebepaling5.Location;
+import android.Manifest;
+import android.app.Activity;
+import android.content.Intent;
+import android.content.IntentSender;
+import android.content.pm.PackageManager;
+import android.location.Location;
+import android.support.v4.app.ActivityCompat;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.google.android.gms.common.ConnectionResult;
+import com.google.android.gms.common.api.GoogleApiClient;
+import com.google.android.gms.common.api.PendingResult;
+import com.google.android.gms.common.api.ResultCallback;
+import com.google.android.gms.common.api.Status;
+import com.google.android.gms.location.LocationListener;
+import com.google.android.gms.location.LocationRequest;
+import com.google.android.gms.location.LocationServices;
+import com.google.android.gms.location.LocationSettingsRequest;
+import com.google.android.gms.location.LocationSettingsResult;
+import com.google.android.gms.location.LocationSettingsStatusCodes;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import stage.hoogtebepaling5.R;
+
+public class LocationClass extends AppCompatActivity implements GoogleApiClient.ConnectionCallbacks,
+        GoogleApiClient.OnConnectionFailedListener,
+        LocationListener{
+
+    private Location myLocation;
+    private GoogleApiClient googleApiClient;
+    private final static int REQUEST_CHECK_SETTINGS_GPS=0x1;
+    private final static int REQUEST_ID_MULTIPLE_PERMISSIONS=0x2;
+    ImageView image;
+    TextView text;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.location_display_layout);
+       // final WifiManager wifi = (WifiManager) getApplicationContext().getSystemService(WIFI_SERVICE);
+        image = findViewById((R.id.mapview));
+        image.setImageResource(R.drawable.beganegrond);
+        text = findViewById(R.id.textView);
+        image.setVisibility(View.INVISIBLE);
```

Comment:
Image kan je beter standaard hiden in de XML maar het gaat toch weg dus geen probleem hier